specification/dashboard/resource-manager/Microsoft.Dashboard/preview/2022-10-01-preview/grafana.json

```diff
@@ -0,0 +1,1998 @@
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results (if there are any).",
+          "readOnly": true
+        }
+      },
+      "description": "A list of private link resources"
+    },
+    "EnterpriseConfigurations": {
+      "description": "Enterprise settings of a Grafana instance",
+      "type": "object",
+      "properties": {
+        "marketplacePlanId": {
+          "description": "The Plan Id of the Azure Marketplace subscription for the Enterprise plugins",
+          "type": "string"
+        },
+        "marketplaceAutoRenew": {
+          "$ref": "#/definitions/MarketplaceAutoRenew"
+        }
+      }
+    },
+    "MarketplaceAutoRenew": {
+      "description": "The AutoRenew setting of the Enterprise subscription",
+      "enum": [
+        "Disabled",
+        "Enabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "MarketplaceAutoRenew"
+      }
+    },
+    "EnterpriseDetails": {
+      "description": "Enterprise details of a Grafana instance",
+      "type": "object",
+      "properties": {
+        "saasSubscriptionDetails": {
+          "$ref": "#/definitions/SaasSubscriptionDetails"
+        },
+        "marketplaceTrialQuota": {
+          "$ref": "#/definitions/MarketplaceTrialQuota"
+        }
+      }
+    },
+    "GrafanaPlugin": {
+      "description": "Plugin of Grafana",
+      "type": "object",
+      "properties": {
+        "pluginId": {
+          "description": "Grafana plugin id",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GrafanaAvailablePlugin": {
+      "description": "Available plugins of grafana",
+      "type": "object",
+      "properties": {
+        "pluginId": {
+          "description": "Grafana plugin id",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Grafana plugin display name",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GrafanaAvailablePluginListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GrafanaAvailablePlugin"
+          },
+          "x-ms-identifiers": [
+            "pluginId"
+          ]
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "SaasSubscriptionDetails": {
+      "description": "SaaS subscription details of a Grafana instance",
+      "type": "object",
+      "properties": {
+        "planId": {
+          "type": "string"
+        },
+        "offerId": {
+          "type": "string"
+        },
+        "publisherId": {
+          "type": "string"
+        },
+        "term": {
+          "$ref": "#/definitions/SubscriptionTerm"
+        }
```

Comment:
Add description to all these properties.

Reply:
[must fix]

Reply:
Will fix in current version